docs/pages/faq/clear-cache-macos-linux.md

```diff
@@ -17,13 +17,13 @@ There are a number of different caches associated with your project that can pre
 
 ## What these commands are doing
 
-If you would like to understand what all that command is doing:
+It is a good habit to understand what commands you find on the internet before you run them.  We break down each commmand below for Expo CLI and Yarn, but the corresponding commands for NPM and React Native CLI have the same function.
```

Comment:
```suggestion
It is a good habit to understand commands you find on the internet before you run them.  We explain each command below for Expo CLI and Yarn, but the corresponding commands for npm and React Native CLI have the same behavior.
```
("Function" and "Behavior" and "Purpose" all work here but the latter are less ambiguous in the context of programming and don't evoke thoughts of code functions.)